docs/sources/flow/setup/install/binary.md

```diff
@@ -45,12 +45,9 @@ To download {{< param "PRODUCT_NAME" >}} as a standalone binary, perform the fol
 
 ## Next steps
 
-- [Start {{< param "PRODUCT_NAME" >}}[Start]
-- [Configure {{< param "PRODUCT_NAME" >}}[Configure]
```

Comment:
I've removed `Configure` link, because there's no configure page for standalone binary. In fact, the instructions on this page already describe how to specify a path to your config file. This is similar to what we have for docker and kubernetes install instructions. We can change this in the future, but I'd rather not do it in this PR.